modules/ROOT/pages/debugging.adoc

```diff
@@ -0,0 +1,18 @@
+= Logging and Debugging
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: debugging, debug, troubleshoot, stack, debugger, using logs, source code, source files, libraries
+
+This section describes how to gather logging information from Mule. It also introduces a number of techniques for troubleshooting Mule issues.
```

Comment:
FYI: I think we're supposed to avoid introductory phrases like "This section describes how to" if we can. Not a big deal.